discord/ext/commands/flags.py

```diff
@@ -277,6 +293,7 @@ class FlagsMeta(type):
         __commands_flag_case_insensitive__: bool
         __commands_flag_delimiter__: str
         __commands_flag_prefix__: str
+        __commands_flag_positional__: Flag | None
```

Comment:
None of the other attributes are typed as optional, could we try to keep that up?

Reply:
My bad for the delay have gotten busy, do you mean as in, using Optional[Flag] or simply using Flag instead ?